PSKoans/Public/Update-PSKoan.ps1

```diff
@@ -0,0 +1,73 @@
+function Update-PSKoan {
+    [CmdletBinding(SupportsShouldProcess, ConfirmImpact = "High",
+        HelpUri = 'https://github.com/vexx32/PSKoans/tree/master/docs/Update-PSKoan.md')]
+    [OutputType([void])]
+    param(
+        [Parameter()]
+        [string[]]
+        $Topic
+    )
+
+    $KoanFolder = Get-PSKoanLocation
+
+    $ModuleKoanFolder = Join-Path -Path $script:ModuleRoot -ChildPath 'Koans'
+    $ModuleKoanList = Get-ChildItem -LiteralPath $ModuleKoanFolder -Recurse -Filter *.Koans.ps1 |
+        Where-Object { -not $Topic -or $_.BaseName -replace '\.Koans$' -in $Topic } |
+        Group-Object { $_.BaseName -replace '\.Koans$' } -AsHashTable -AsString
+
+    $UserKoanList = Get-ChildItem -LiteralPath $KoanFolder -Recurse -Filter *.Koans.ps1 |
+        Where-Object { -not $Topic -or $_.BaseName -replace '\.Koans$' -in $Topic } |
+        Group-Object { $_.BaseName -replace '\.Koans$' } -AsHashTable -AsString
+
+    $TopicList = [System.Collections.Generic.HashSet[String]]::new([System.StringComparer]::InvariantCultureIgnoreCase)

```

Comment:
```suggestion
    $TopicList = [HashSet[String]]::new(
        [System.StringComparer]::InvariantCultureIgnoreCase)
```
Also, make sure you add the appropriate `using namespace` to the top of the file. 🙂